Source/Serilog.Exceptions/Filters/IExceptionPropertyFilter.cs

```diff
@@ -0,0 +1,22 @@
+namespace Serilog.Exceptions.Filters
+{
+    using System;
+
+    /// <summary>
+    /// Interface used for filtering exception properties.
+    /// Filtering process is global, each property of every exception will
+    /// go through a configured exception property filter.
+    /// </summary>
+    public interface IExceptionPropertyFilter
+    {
+        /// <summary>
+        /// Called after the property was discovered and destructured but
+        /// just before it is added to results.
+        /// </summary>
+        /// <param name="exceptionType">Type of the exception for which properties are filtered</param>
+        /// <param name="propertyName">Name of the property</param>
+        /// <param name="value">Destructured value of the property </param>
+        /// <returns>Boolean flag indicating whether property will be rejected</returns>
+        bool ShouldPropertyBeFiltered(Type exceptionType, string propertyName, object value);
```

Comment:
I still like `CanAddProperty`

Reply:
Could you elaborate a little bit? I am not saying that is a bad name, it's just my gut feeling that is better to phrase the method so that it specifies properties that should be excluded(like `ShouldBeFiltered` or `ShouldBeRejected`) then for it to specify properties to be added(like CanAddProperty or ShouldBeAdded).